ROADMAP.md

```diff
@@ -1,37 +1,32 @@
 # Hubot Roadmap
```

Comment:
I think it can be safe to remove this file to reflect there isn't one currently 🙊

Reply:
Sounds good.

Reply:
I've removed it. I'm guessing we'll want to create a new one.